ai/cryptocom-ai-agent-chatbot/src/integration/chain-ai.api.ts

```diff
@@ -32,6 +32,7 @@ export const chainAiInstance = {
           openAI: {
             apiKey: import.meta.env.VITE_OPEN_AI_KEY,
           },
+          chain: 240
```

Comment:
do we want to pass chainId here? Or are we already setting the chainId in the ai-agent-service?

Reply:
not necessary, removed